crawler/main.py

```diff
@@ -53,6 +56,19 @@ def run(sftp: bool, keep_files: bool, add_to_dart: bool, settings_module: str =
 
             # imports_collection = get_mongo_collection(db, COLLECTION_IMPORTS)
```

Comment:
This comment could be removed

Reply:
I didn't even add that comment 😆. Have removed